storage/allocator_test.go

```diff
@@ -58,32 +60,45 @@ var multiDCConfig = config.ZoneConfig{
 // synchronize on their callbacks. There can only be one storeGossiper used per
 // gossip instance.
 type storeGossiper struct {
-	g  *gossip.Gossip
-	wg sync.WaitGroup
-	mu sync.Mutex
+	g           *gossip.Gossip
+	wg          sync.WaitGroup
+	mu          sync.Mutex
+	storeKeyMap map[string]bool
```

Comment:
We prefer `map[string]struct{}` to `map[string]bool` for a set, so there can be no confusion about what it means when `v, ok := m[k]` returns `false, true`.